chalice/package.py

```diff
@@ -357,15 +386,255 @@ def _register_cfn_resource_name(self, name):
         return cfn_name
 
 
+class TerraformGenerator(TemplateGenerator):
+
+    template_file = "chalice.tf.json"
+
+    def generate(self, resources):
+        # type: (List[models.Model]) -> Dict[str, Any]
+        template = {
+            'resource': {},
+            'terraform': {
+                'required_version': '> 0.11.0'
```

Comment:
Also what's your thoughts on setting the minimum version to the latest major version of troposphere `0.12`?

Reply:
i'm notionally fine with it, but this integration was tested with 0.11 and works seamlessly across that boundary. historically i witnessed many versions of tf extant in a single org, so aiming for broader reach where it is known compatible.

Reply:
to be honest given the shared ci situation at a lot of large orgs and that has been tested with 0.11 I'd rather leave as is, the 0.12 shift was fairly large but its also fairly recent, and redress start of q4.

Reply:
Sounds good. We can keep it at `0.11` then.